reconcile/utils/oc.py

```diff
@@ -147,23 +154,28 @@ def __init__(self, namespace, resource, server,
         self.is_log_slow_oc_reconcile = is_log_slow_oc_reconcile
 
 
-class OC:
+class OCDeprecated:
     def __init__(self, server, token, jh=None, settings=None,
                  init_projects=False, init_api_resources=False,
                  local=False):
         self.server = server
         oc_base_cmd = [
             'oc',
-            '--kubeconfig', '/dev/null',
-            '--server', server,
-            '--token', token
+            '--kubeconfig', '/dev/null'
         ]
+        if server is not None and len(server) > 0:
```

Comment:
I find `if server:` shorter and easier to read. In English, to me it reads as "if there is a server".

Reply:
+1 to @Piojo's suggestion.

@rrati I prefer changing the default to `None` as you did, but is there a reason for this? or are you fixing this purely because of style and legibility?

Reply:
Namely fixing for legibility and consistency with error handling for other empty things.

Reply:
Make change suggested by @Piojo